src/OrleansCodeGenerator/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "Microsoft.CodeAnalysis.CSharp": "1.0.0"
+    "Microsoft.CodeAnalysis.CSharp": "1.3.2"
```

Comment:
can't it be solved with binding redirects on the benchmark project? Seems like a pretty big user facing update that we should be able to avoid.
Otherwise we'd have to update it everywhere, including the nuspec files